### DISCUSSION TITLE
怎么开启 mako 编译，在配置文件种增加mako: {} 不起作用

### DISCUSSION THREAD

**Discussion by lanteng007** [umijs/umi#12640] [Q&A]
怎么开启 mako 编译，在配置文件种增加mako: {} 不起作用

**Comment by lanteng007** [comment]
![image](https://github.com/user-attachments/assets/cacf7d21-3097-4d49-ab7b-c68d2ae00133)

**Comment by lanteng007** [comment]
![image](https://github.com/user-attachments/assets/3ddc4a94-4f81-4b35-83c5-561ef68a44e2)

**Comment by xiaohuoni** [comment]
umi 版本？

**Reply by lanteng007** [reply]
v4.3.12

**Reply by xiaohuoni** [reply]
抱歉，刚刚图片没刷出来。这个版本看起来没有问题
[umi-mako.zip](https://github.com/user-attachments/files/16622611/umi-mako.zip)

**Reply by lanteng007** [reply]
![image](https://github.com/user-attachments/assets/5b6a7af0-3ba2-4b77-a42f-1b6dd207d9c5)
用你给的试也是一样的

**Reply by lanteng007** [reply]
是有什么特殊环境要求么，node版本是18.18
